.circleci/config.yml

```diff
@@ -53,6 +53,8 @@ jobs:
             git add src/201901-CDP/content.json
             git config user.email "support@spry-group.com"
             git config user.name "Spry Bot"
+            npm run build
```

Comment:
I think you need to run both npm update-data and npm build.

Reply:
That already happens in a previous step